ci-operator/step-registry/baremetal/lab/agent/install/network/static/ephemeral/baremetal-lab-agent-install-network-static-ephemeral-workflow.yaml

```diff
@@ -1,6 +1,8 @@
 workflow:
   as: baremetal-lab-agent-install-network-static-ephemeral
   steps:
+    env:
+      VENDOR: dell
```

Comment:
Mmm, I wouldn't do this here as it's a bit hidden and will require a different setup with the explicit vendor only for the arm64 jobs (hpe)

Reply:
Moved to https://github.com/openshift/release/pull/46996/files#diff-50397b8e1c037c03c575bb66443685d531f5333a1885375f94d7538e62e82591R1918 and